src/utils/WASDPlusControls.js

```diff
@@ -0,0 +1,207 @@
+import AFRAME from "aframe";
+import * as THREE from "three";
+let bind = AFRAME.utils.bind;
+let shouldCaptureKeyEvent = AFRAME.utils.shouldCaptureKeyEvent;
+
+let KEYCODE_TO_CODE = {
+    "38": "ArrowUp",
+    "37": "ArrowLeft",
+    "40": "ArrowDown",
+    "39": "ArrowRight",
+    "87": "KeyW",
+    "65": "KeyA",
+    "83": "KeyS",
+    "68": "KeyD",
+    "32": "Space",
+    "16": "ShiftLeft"
+};
+
+let KEYS = [
```

Comment:
This should be const. It should also probably be generated from the above object.